### ISSUE TITLE
UI designs: add Test Plan Reports that require a specific AT Version

### ISSUE THREAD

**Issue by ccanash** [w3c/aria-at-app#952]
#791 objective describes needing a specific AT version to be collected when adding a new entry into the Test Queue. The version could be marked as "Any recent" or is any known AT Versions already tracked in the system.

Note that if a test plan is already in the recommended phase, the admin cannot select "Any recent", only being able to specify a version.

**Comment by isaacdurazo** [comment]
Here are the mockups with descriptions of the changes to address issues #791 and #792.

## Test Queue

Under the Add Test Plans disclosure, a dropdown to select an Assistive Technology version has been added next to the one to select an AT.

There is also a new column in each table for AT and Browser version requirements. The cells under this column will display the AT and Browser's names along with their version as needed.

![ARIA-AT - Test Queue](https://github.com/w3c/aria-at-app/assets/1379244/43fa6ead-6832-4cd0-9e1d-05456e01e52f)

## Data Management

The only visual change to the Data Management page is under the existing "Covered AT" column, which now includes version numbers for the ATs listed.

![ARIA-AT - Data Management](https://github.com/w3c/aria-at-app/assets/1379244/0e57f5fa-3055-4017-97f1-9a0b3f5a51ae)

## Test Plan report status dialog

The only visual change to the Test Plan report status dialog is under the AT column, which now includes version numbers for the ATs listed.

![ARIA-AT - Data Management – Test plan report status dialog](https://github.com/w3c/aria-at-app/assets/1379244/b558a87d-9012-4124-b491-5e9bec3cffa7)

**Comment by mcking65** [comment]
@isaacdurazo @ccanash 

First one simple change: We do not need to update the covered AT column on data management because each row of the data management table is for all versions of a test plan; it is not specific to any AT version. It is simply describing which AT have commands in any version of the test plan.

We have two major open issues that need to be resolved before we can finalize this design.

First, when I first raised #791, I had not yet considered the kind of situation caused by changes to AT commands and settings like the recent change Apple made to VoiceOver commands for quick nav settings. That was the inspiration for w3c/aria-at#1047. That issue has thus inspired me to think that the AT version requirement in draft review should be more specific than "Any Recent". I have suggested an alternative in [this comment in 791](https://github.com/w3c/aria-at-app/issues/791#issuecomment-2002545439), which I will discuss with the PAC team on Tuesday.

Second, I am strongly leaning against investing any more in making changes to the current test queue page tables. The tables on the test queue page today are quite problematic, and pouring anything more into them seems like an inefficient use of resources. I think we need to evaluate the [option B proposal in this comment on 791](https://github.com/w3c/aria-at-app/issues/791#issuecomment-2002610093).
